.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,14 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Is this file usually checked in? I don't use this editor but if it helps other people who do, I'm happy for it to be included.

Reply:
Yes: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems

Which editor do you use?

Reply:
@leventov cool, sounds good then! I use Atom for now :)